rosa_hcp/rosa-hcp-egress-lockdown-install.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-egress-lockdown-install"]
+= Creating a {product-title} cluster with egress lockdown
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-egress-lockdown-install
+toc::[]
+
+You can create a fully operational cluster that does not require a public egress by configuring a virtual private cloud (VPC) and using the `--properties zero_egress:true` flag when creating your cluster.
+
+.Prequisites
+
+* You have an AWS account with sufficient permissions to create VPCs, subnets, and other required infrastructure.
```

Comment:
Also the prereqs for default security groups and vpc privatelink connections setup for AWS services. See: https://access.redhat.com/articles/7090154#create-the-vpc-via-aws-consolecli-4